src/Compilers/CSharp/Portable/Symbols/Symbol_Attributes.cs

```diff
@@ -624,6 +624,21 @@ private void RecordPresenceOfBadAttributes(ImmutableArray<CSharpAttributeData> b
             }
         }
 
+        /// <summary>Binds attributes without decoding known attributes (assumes this has already been done).</summary>
+        /// <param name="binderOpt">Binder to use. If null, <see cref="DeclaringCompilation"/> GetBinderFactory will be used.</param>
+        protected BoundAttribute[] BindAttributes(OneOrMany<SyntaxList<AttributeListSyntax>> attributeDeclarationSyntaxLists, Binder binderOpt)
+        {
+            var attributesToBind = GetAttributesToBind(attributeDeclarationSyntaxLists, AttributeLocation.None, BindingDiagnosticBag.Discarded, DeclaringCompilation, attributeMatchesOpt: null, binderOpt, out var binders);
```

Comment:
>GetAttributesToBind [](http://example.com/codeflow?start=35&length=19)

It looks like this is going to filter some attributes out. Do we really want this behavior? Regardless, we probably should have a test covering scenarios when GetAttributesToBind filters out an attribute.

Reply:
Right, thanks. I was trying to replicate `LoadAndValidateAttributes` but I don't longer use `GetAttributesToBind`: 076be8220f4b65b221918b7e666ca5e3393e786c